tests/contrib/hooks/test_spark_submit_hook.py

```diff
@@ -364,7 +365,8 @@ def test_resolve_connection_spark_home_not_set_connection(self):
 
     def test_resolve_connection_spark_binary_set_connection(self):
         # Given
-        hook = SparkSubmitHook(conn_id='spark_binary_set')
+        hook = SparkSubmitHook(conn_id='spark_binary_set',
+                               spark_binary='custom-spark-submit')
```

Comment:
This test should pass without this change (as the spark binary was set in the connection)

We need a _new_ test that passes something _other_ than `custom-spark-binary` here and make sure that is used instead.